### DISCUSSION TITLE
How to update Security Onion Intrusion Detection Honeypot (SO IDH)?

### DISCUSSION THREAD

**Discussion by linuxlurak** [Security-Onion-Solutions/securityonion#12388] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

2

### RAM

16GB

### Storage for /

100

### Storage for /nsm

lots of GB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Is there need to keep  Security Onion Intrusion Detection Honeypot (SO IDH) up to date? 

I tried to run soup on my honeypot which fails with `Failed to start salt-master.`. As it's only a honeypot this seems  to me to be normal behavior. 

But if we need to update IDH, how to do this? And hwo do we know if updating is necessary?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by robbiemarshall** [comment]
The IDH node (and all other minion nodes) will be updated when you run soup on your manager.
